apps/web/src/components/Layout/MaintenanceBanner.tsx

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable import/no-duplicates */
+import React from "react";
+import { useIntl } from "react-intl";
+import isAfter from "date-fns/isAfter";
+import isBefore from "date-fns/isBefore";
+import isEqual from "date-fns/isEqual";
+
+import { formatDate, parseDateTimeUtc } from "@gc-digital-talent/date-helpers";
+import { getRuntimeVariable } from "@gc-digital-talent/env";
+import { Alert } from "@gc-digital-talent/ui";
+
+const MaintenanceBanner = () => {
+  const intl = useIntl();
+  // Outage time range, date, starttime
+  const maintenanceBannerPublicDate = getRuntimeVariable(
+    "MAINTENANCE_BANNER_PUBLIC_DATE",
+  );
+  const serverMaintenanceDate = getRuntimeVariable(
+    "MAINTENANCE_BANNER_START_DATE",
+  );
+  const serverMaintenanceDuration = getRuntimeVariable(
+    "MAINTENANCE_BANNER_DURATION",
+  );
+
+  const startDate = serverMaintenanceDate
+    ? formatDate({
+        date: parseDateTimeUtc(serverMaintenanceDate),
+        formatString: `PPPPpp`,
+        intl,
+      })
+    : null;
+
+  const duration =
+    serverMaintenanceDuration && !!Math.abs(+serverMaintenanceDuration)
+      ? Math.abs(+serverMaintenanceDuration)
+      : null;
+
+  const showMaintenanceBanner =
```

Comment:
I think there's a bug here.  I'm before the public date but it's still showing.
Also, the message disappears when the current date is the same as the public date.  Maybe it's backwards?
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/e7909b5a-fbaf-4d05-a06d-fd71b8c7e973)

Reply:
Ya the values should be reversed 😓 Fixed by [d85c93e](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7832/commits/d85c93ea09b93ddc6d8cc92a0702a10051a6455e)